.github/workflows/release.yml

```diff
@@ -49,11 +50,11 @@ jobs:
         uses: gmiam/rust-musl-action@9e6a37bf27ecfffb6b92240ea276bea5487fa15d
         continue-on-error: false
         with:
-          args: cargo install cargo-deb && cargo deb && rustup target add $BUILD_TARGET
+          args: cargo install cargo-deb && cargo deb
       - uses: actions/upload-artifact@v2
         with:
-          name: fpm.deb
-          path: target/debian/${{ env.BINARY_NAME }}*
+          name: fpm_debian
+          path: target/release/${{ env.BINARY_NAME }}*
```

Comment:
This is wrong. I delete the env.BINARY_NAME variable, why are you bring it back? You are confused. The name of the file generated by `cargo deb` is not `fpm` as you have written right now, it's `fpm.deb`. It is matching right now because you have used `${{ env.BINARY_NAME }}*` but that's making things confusing. Just put the path explicitly and delete the variable. Do not upload more than needed (ask yourself do should this action upload only fpm.deb file or every file in target folder that starts with `fpm` as you have written?

Reply:
I have given the full path name now in the latest commit. When I remove the binary name, which is being used by fpm controller,
**"          name: fpm_controller_linux_musl_x86_64
          path: target/x86_64-unknown-linux-musl/release/${{ env.BINARY_NAME }}*"**
          
it gives error -   
**"mv ~/download/fpm_controller_linux_musl/fpm.d ~/download/fpm_controller_linux_musl/fpm_controller_linux_musl_x86_64.d
  shell: /usr/bin/bash -e {0}
mv: cannot stat '/home/runner/download/fpm_controller_linux_musl/fpm.d': No such file or directory"**